openstack/orchestration/v1/apiversions/requests.go

```diff
@@ -0,0 +1,13 @@
+package apiversions
+
+import (
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+// ListVersions lists all the Neutron API versions available to end-users
+func ListVersions(c *gophercloud.ServiceClient) pagination.Pager {
```

Comment:
Though I don't think it _would_ change, the API for `apiversions` could change in a later version of the Orchestration API. Also, we've already started doing it like this: `networking` and `blockstorage` have similiar `apiversions` packages.